com.github.watheusctavares.minecraftlauncher.yml

```diff
@@ -0,0 +1,55 @@
+app-id: com.github.watheusctavares.minecraftlauncher
+runtime: org.freedesktop.Platform
+runtime-version: '25.08'
+sdk: org.freedesktop.Sdk
+command: minecraft-launcher
+tags:
+  - proprietary
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=system-bus
+  - --persist=.minecraft
+
+modules:
+  - name: minecraft-launcher
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://launcher.mojang.com/download/Minecraft.tar.gz
```

Comment:
this is unlikely to be legally redistributable. It belongs to Mojang.

Also it's not versioned so it will randomly break when updated.

At best `extra-data` should be used. But I don't see where the "launcher" is.